lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+  attr_reader :dishes
+  def initialize
+    @dishes = [
+      {number: 1, dish: 'Chicken Biryani', price: 8.99},
```

Comment:
To make the code cleaner you could you a Dish class, with @name and @price as instance parameters.